### DISCUSSION TITLE
How to override styles for an existing Tailwind website?

### DISCUSSION THREAD

**Discussion by martinszeltins** [tailwindlabs/tailwindcss#8329] [Help]
If there is an existing 3rd party / external Tailwind website where I cannot change HTML or CSS, what is the best practice to override it's styles?

The problem is that there are no easy selectors like `.button` or `.sidebar` but everything is written with many utility classes. My solution has been to target those utility classes but I wonder if there is a better way.

Example Tailwind site:

````
<body class="bg-gray-200 p-5 text-gray-800">
  <div class="font-semibold text-gray-900 hover:text-gray-800 hover:bg-gray-100">
    Hello, world!
  </div>
</body>
````

In order to change these styles I need to write something like this:

````
.bg-gray-200 {
  background-color: #330022;
}

.text-gray-900 {
  background-color: #ff0345;
}

.hover\:text-gray-800:hover {
  color: green;
}
````

Is there a better alternative?

**Comment by carlos-dubon** [comment]
No, as you've already mentioned it this is how Tailwind works and that's the only way to change its CSS styles.
